app/controllers/aboutYourLeaseOrTenure/AboutYourLandlordController.scala

```diff
@@ -51,6 +53,26 @@ class AboutYourLandlordController @Inject() (
     with Logging {
 
   def show: Action[AnyContent] = (Action andThen withSessionRefiner).async { implicit request =>
+    val containCYA = request.uri
+    val forType    = request.sessionData.forType
+
+    containCYA match {
+      case containsCYA if containsCYA.contains("=CYA") =>
+        audit.sendExplicitAudit("cya-change-link", ChangeLinkAudit(forType.toString, request.uri, "AboutYourLandlord"))
+      case _                                           =>
+        Future.successful(
+          Ok(
+            aboutYourLandlordView(
+              request.sessionData.aboutLeaseOrAgreementPartOne.flatMap(_.aboutTheLandlord) match {
+                case Some(aboutTheLandlord) => aboutTheLandlordForm.fill(aboutTheLandlord)
+                case _                      => aboutTheLandlordForm
+              },
+              request.sessionData.toSummary,
+              getBackLink(request.sessionData)
+            )
+          )
+        )
+    }
```

Comment:
We are now calling the aboutYourLandlordView twice inside of show(). This comment applies to others controllers too.